lib/chef/web/core/helpers.rb

```diff
@@ -0,0 +1,34 @@
+require 'chef/web/core'
+require 'chef/web/core/component'
+require 'nokogiri'
+
+class Chef
+  module Web
+    module Core
+      module Helpers
+
+        def chef_logo(element=:div, opts={})
```

Comment:
In Rails, I had to do

```erb
<%= chef_logo.html_safe %>
```
not sure if we can make it do that by default and have it work in both Rails and Middleman.